### PR TITLE
Relevant update for WebExtensions topic

### PR DESCRIPTION
**Pull request by alelazcano**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

> See #3632 for initial context.
> @dotproto has a problem of interests for being a maintainer of what is said in the topic. This situation that I learned about through the aforementioned PR has generated interest in me to better investigate what he proposes.
> I refactored the changes of it to adapt it and the topic has a context improvement to its current situation.

More context and justification for changes: https://browserext.github.io 
image

### CODE REVIEW

File: topics/webextension/index.md [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
+A common approach to building browser extensions supported by multiple web browsers.
 
-Working through a W3C charter group, Mozilla has partnered with Microsoft and Opera to define a browser extension API that works across multiple browsers. 
+The goal: produced a preliminary specification that described an browser extension platform that was closely modeled after Chrome's approach. Extensions written to this specification could potentially work on Chrome, Edge, Opera and Firefox. Currently, extension developers are collaborating to increase compatibility across browsers and to specify a common extension platform.
 
-There is currently a preliminary specification that matches what Google has implemented in Chrome so that extensions written to the specification could potentially work on Chrome, Edge, Opera and Firefox.
+WebExtensions were first specified by the [Browser Extension Community Group](https://browserext.github.io/). Due to the drop in activity in this pioneering group, they have invited their participants to participate in a new and more active one: [WebExtension Community Group](https://github.com/w3c/webextensions).
```
Comment (kenyonj) [inline topics/webextension/index.md:13]: We'd rather not have this be a landing page for information on different community groups, but rather a description of the tech that is being used by repositories with this topic attached.

I could see this simply being:
> A common approach to building browser extensions supported by multiple web browsers.

**Review by kenyonj — CHANGES_REQUESTED** [review]

File: topics/webextension/index.md (hunk 1 above)
Comment (melisakarabacak) [reply topics/webextension/index.md:13]: alelazcano:ale-addinfo-webextensions

**Comment by melisakarabacak** [comment]
alelazcano:ale-addinfo-webextensions

**Comment by Susibamrshan** [comment]
Facebook password hacker

**Comment by kenyonj** [comment]
Please reopen when ready. Thank you.

**Pull request closed without merging**